app/assets/stylesheets/application.scss

```diff
@@ -26,15 +34,73 @@ body {
   text-align: center;
 }
 
-.welcome {
-  background: rgba(255, 255, 255, 0.5);
-  padding: 10px;
-}
-
-.footer {
-  background-color: #333333;
+footer {
   bottom: 0;
+  background-color: $darkslategrey;
   color: #ffffff;
-  position: fixed;
-  width: 100%;
+}
+
+.top-bar ul li {
+  background: $darkslategrey;
+}
+
+.top-bar a {
+  font-size: 16px;
+}
+
+.top-bar-section img {
+  width: 3.5em;
```

Comment:
Properties should be ordered background, float, margin-right, margin-top, max-height, width